app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,29 @@
+class PostsController < ApplicationController
+  require "will_paginate/array"
+  PAGE_SIZE = 4
+
+  def index
+    page = (params[:page] || '1').to_i
+
+    client = Tumblr::Client.new({
```

Comment:
Perhaps they could bypass git and just edit the text on github? If they're
comfortable working with templates at all, that is

On Tue, Dec 22, 2015 at 3:27 PM William Jeffries <notifications@github.com>
wrote:

> In app/controllers/posts_controller.rb
> <https://github.com/heatseeknyc/heatseeknyc/pull/39#discussion_r48276431>:
>
> > @@ -0,0 +1,29 @@
> > +class PostsController < ApplicationController
> > +  require "will_paginate/array"
> > +  PAGE_SIZE = 4
> > +
> > +  def index
> > +    page = (params[:page] || '1').to_i
> > +
> > +    client = Tumblr::Client.new({
>
> Yeah static was the wrong word there. I guess non-programmatic. We're
> talking about the content that any normal business would put on their
> website, like their address and the names of their employees. Everything
> that doesn't have to do with the temperature monitoring app.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/heatseeknyc/heatseeknyc/pull/39/files#r48276431>.
>